### ISSUE TITLE
Add support for overriding scoped imports

### ISSUE THREAD

**Issue by dominiktopp** [single-spa/import-map-overrides#110]
Hi,

import-maps also allow scoped mappings (https://github.com/systemjs/systemjs/blob/main/docs/import-maps.md#scopes).
At the moment I do not find a way to override such an import. Would it be possible to add this to import-map-overrides?

Thank you
Dominik

**Comment by joeldenning** [comment]
I agree this would be a good feature to add. Would you be willing to fund the development of it? I charge $250/hr, billed to the minute.
